Automattic/delphin
# Add SASS linting

This adds SASS linting. The behaviour and layout on all pages should be the same
#### Testing instructions
1. Run `git checkout add/sass-linting` and start your server, or open a [live branch](https://delphin.live/?branch=add/sass-linting)
2. Check the layout of all pages, but especially notices, the form toggle on checkout, the contact information form, my domains and the progress bar.

@Tug - I removed some of the transition `all` from My Domains as I couldn't see what function they served. It wasn't possible to test because My Domains is broken in master...
#### Reviews

@Automattic/sdev-feed
